src/test/groovy/de/qaware/gradle/plugin/xsd2java/xjc/Xsd2JavaPluginTest.groovy

```diff
@@ -19,8 +19,8 @@ import de.qaware.gradle.plugin.xsd2java.xjc.tasks.XSD2JavaTask
 import org.gradle.api.Task
 import org.gradle.testfixtures.ProjectBuilder
 import org.gradle.testkit.runner.GradleRunner
-import org.junit.Rule
-import org.junit.rules.TemporaryFolder
+import org.gradle.internal.impldep.org.junit.Rule
```

Comment:
Don' think that depending on a Gradle internal API is such a good idea. Is there another option?

Reply:
Hi @johnpooley! In org.spockframework:spock-core:2.0-groovy-3.0 deleted junit. What do you think about add dependency testImplementation 'junit:junit:4.13.2' and revert changes into Xsd2JavaPluginTest?

Reply:
Already working on the PR. 
The solution is to use the following dependency: `testImplementation 'org.spockframework:spock-junit4:2.0-groovy-3.0'`

Reply:
hello, just reading this now. Added this change to the PR here too